src/css-grids/grids.html

```diff
@@ -0,0 +1,38 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>CSS Grids</title>
+    <link rel="stylesheet" href="./styles.css">
+    
+</head>
+<body>
+    <div class="grid-container">
+        <div class="box red-box">
+            <h1>A DIGITAL CREATIVE AGENCY</h1>
+            <h3>we work on a diverse range of projects, from helping startups get their buisness up.</h3>
+        </div>
+        <div class="box blue-box">
+            <h3>Featured</h3>
+            <h1>PROJECT</h1>
+        </div>
+        <div class ="box yellow-box">
+            </div>
```

Comment:
put this closing div tag on the same line. 
`<div class ="box yellow-box"></div>`